tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = py38-django{32}, quality-django{32}
+envlist = py38, quality
```

Comment:
I'm not sure what benefit they would provide -- tox would no longer be controlling the Django version, and still wouldn't be once 4.x testing starts.

Reply:
It would be for local testing of the 4.x upgrade.  But I suppose it would be clearer to remove this for now and just add it back once it's actually implemented when we start that testing.

Reply:
I think for local testing what we'd want is just a branch where Django is upgraded to 4.x and tests are otherwise run as normal.